Makefile

```diff
@@ -1,14 +1,14 @@
 .PHONY: clean dev all check checkstatic unittests test phpdev javascript
 
 all: VERSION javascript
-	composer install --no-dev
+	composer install --no-dev --prefer-dist
```

Comment:
This can prevent caching errors on Travis. It'll try to download entire GitHub repositories unless this flag is set.

Reply:
What "caching errors"? We shouldn't be changing our build process for a dubious optimization.

Reply:
Sorry for the imprecise comment there.

This is what the flag does

>--prefer-dist: Reverse of --prefer-source, Composer will install from dist if possible. **This can speed up installs substantially on build servers** and other use cases where you typically do not run updates of the vendors. It is also a way to circumvent problems with git if you do not have a proper setup.
 https://getcomposer.org/doc/03-cli.md (emphasis mine)

I saw it recommended somewhere that this option is good to set because it can reduce the Travis start up time.